sample/src/main/java/org/smartregister/simprint/sample/MainActivity.java

```diff
@@ -23,7 +23,8 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        SimPrintsLibrary.init(MainActivity.this,"tZqJnw0ajK04LMYdZzyw","test_user");
+//        SimPrintsLibrary.init(MainActivity.this,"tZqJnw0ajK04LMYdZzyw","test_user");
+        SimPrintsLibrary.init(MainActivity.this,"fUBnpzDdbsCsMp0egCHB","global_module");
```

Comment:
It would be good to move this to local.properties

Reply:
Okay, that makes sense. I have moved it already

Reply:
I have also updated the sample app to include the identification piece.